Jenkinsfile

```diff
@@ -43,6 +43,13 @@ def webTestUrlParameter() {
     return env.WEB_TEST_URL?.isEmpty() ? '' : "-PwebTestUrl='${params.WEB_TEST_URL}'"
 }
 
+def adMobParameters() {
+    return !env.ADMOB_BANNER_APP_ID?.isEmpty() && !env.ADMOB_BANNER_UNIT_ID?.isEmpty() ?
+            "-PadMobBannerAppId='${env.ADMOB_BANNER_APP_ID}' -PadMobBannerUnitId='${env.ADMOB_BANNER_UNIT_ID}'"
+            : "-PadMobBannerAppId=ca-app-pub-5619801003513533~9206664779 " +
+            "-PadMobBannerUnitId=ca-app-pub-5619801003513533/1184704529"
```

Comment:
How did we create these test ids?

Reply:
they are build params, see [jenkins build](https://jenkins.catrob.at/job/Catroid/job/PR-3917/36/parameters/)